cdc/sink/dmlsink/mq/manager/pulsar_manager.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2023 PingCAP, Inc.
```

Comment:
It seems that we don't need a pulsar topic manager at all.

Reply:
can we just remove this file?

Reply:
An empty implementation can be given, but it cannot be deleted, otherwise the file mq_ddl_sink.go will not work